vincentlanglet/twig-cs-fixer/0.6/manifest.json

```diff
@@ -1,5 +1,8 @@
 {
     "gitignore": [
         "/.twig-cs-fixer.cache"
-    ]
+    ],
+    "copy-from-recipe": {
+        ".twig-cs-fixer.php": ".twig-cs-fixer.php"
```

Comment:
I dunno if we should provide this default config file.

Without config file, I believe my tool is already using `TwigCsFixer` standard by default. Which means that if you add this config file like this without changing anything in it, it's just a useless file.

So maybe the post-install message is enough ?

Reply:
Since it can work without the config file, I don't know if it's a good idea to add automatically a config.
I'm unsure how other tools work (and if they can work without a config file)...

But the `post-install.txt` is great

Reply:
Ah i think its a good addition as you can indicate you can make changes to the config like adding more rule, if you dont change anything you still get the same results. So i'm for it.

As for others i see php-cs-fixer has config: https://github.com/symfony/recipes/blob/main/friendsofphp/php-cs-fixer/3.0/.php-cs-fixer.dist.php but this doesn't work without it i assume